test/release/examples/primers/associative.good

```diff
@@ -1,9 +1,9 @@
-associative.chpl:72: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:306: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:307: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:309: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:314: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
-associative.chpl:315: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
+associative.chpl:68: warning: whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)
```

Comment:
See the [sort PREDIFF](https://github.com/chapel-lang/chapel/blob/master/test/modules/packages/Sort/errors/PREDIFF) as an example of removing these line numbers.